src/perturbopy/yml_files/input_parameters_perturbo.yml

```diff
@@ -95,7 +95,7 @@ debug:
 yaml_fname:
    family: job_control
    type: string
-   default: "'.yml'"
+   default: "'pert_output.yml'"
```

Comment:
no no, this is an error!
Read the description below.
If the name is '.yml', then the output will be si_setup.yml or si_dynamics.yml, this is very important to not mix the YAML files, please change back :)